extensions/ql-vscode/src/packaging.ts

```diff
@@ -82,26 +83,44 @@ interface QLPackQuickPickItem extends QuickPickItem {
 export async function handleInstallPackDependencies(
   cliServer: CodeQLCliServer,
   progress: ProgressCallback,
+  relativeDirectoryPath?: string,
 ): Promise<void> {
   progress({
     message: "Choose packs to install dependencies for",
     step: 1,
     maxStep: 2,
   });
-  const workspacePacks = await cliServer.resolveQlpacks(
-    getOnDiskWorkspaceFolders(),
-  );
-  const quickPickItems = Object.entries(
-    workspacePacks,
-  ).map<QLPackQuickPickItem>(([key, value]) => ({
-    label: key,
-    packRootDir: value,
-  }));
-  const packsToInstall = await window.showQuickPick(quickPickItems, {
-    placeHolder: "Select packs to install dependencies for",
-    canPickMany: true,
-    ignoreFocusOut: true,
-  });
+
+  let packsToInstall: QLPackQuickPickItem[] | undefined;
+
+  if (relativeDirectoryPath) {
+    const dirPath = resolve(
+      join(workspace.rootPath || "", relativeDirectoryPath),
+    );
+
+    packsToInstall = [
+      {
+        label: relativeDirectoryPath,
+        packRootDir: [dirPath],
+      },
+    ];
+  } else {
+    const workspacePacks = await cliServer.resolveQlpacks(
+      getOnDiskWorkspaceFolders(),
+    );
+    const quickPickItems = Object.entries(
```

Comment:
In the case of a code tour, do we want to install the packs automatically without a popup?